notebooks/RAG_zephyr_langchain.ipynb

```diff
@@ -0,0 +1,11589 @@
+{
```

Comment:
<p>Maybe clear this output since it's not really informative?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>